.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
         - id: black
           language_version: python3
       exclude: ^tests/
-  -   repo: https://gitlab.com/pycqa/flake8
+  -   repo: https://github.com/pycqa/flake8
```

Comment:
When I use pre-commit, the repo askes for my gitlab account. Change host to github fixed this issue